src/bin/spin.rs

```diff
@@ -27,18 +28,17 @@ fn version() -> &'static str {
 }
 
 /// The Spin CLI
-#[derive(Debug, StructOpt)]
-#[structopt(
+#[derive(Parser, Debug)]
+#[clap(
     name = "spin",
     version = version(),
```

Comment:
my favorites you can have if you like them: 

```
#[clap(author, version, about, long_about = None)]
#[clap(propagate_version = true)]
```

Reply:
I'm just shooting for a straight upgrade in this PR.

Reply:
SHIPIT